From e54df903f8558a9d966b53de421ffc026bc92c1f Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 2 Sep 2024 07:04:19 +0200
Subject: [PATCH] watchcat: Safer defaults

Do not restart router just because google is down. In fact do not
restart the whole router by default, but just an interface if none of the
three public DNS servers responds.
---
 utils/watchcat/files/watchcat.config | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/watchcat/files/watchcat.config b/utils/watchcat/files/watchcat.config
index ed6544cc3..2ee779cb6 100644
--- a/utils/watchcat/files/watchcat.config
+++ b/utils/watchcat/files/watchcat.config
@@ -1,5 +1,5 @@
-config watchcat
+config watchcat "default"
 	option period '6h'
-	option mode 'ping_reboot'
-	option pinghosts '8.8.8.8'
+	option mode 'restart_iface'
+	option pinghosts '1.1.1.1 8.8.8.8 9.9.9.9'
 	option forcedelay '30'
-- 
2.46.0

